kcli_pre.sh

```diff
@@ -134,3 +134,18 @@ echo argocd_clusters_app_path needs to be set for argocd && exit 1
 echo argocd_policies_app_path needs to be set for argocd && exit 1
 {% endif %}
 {% endif %}
+
+## Cleaning
+CLUSTER={{ cluster }}
+POOL={{ pool }}
+POOLPATH=$(kcli list pool | grep $POOL | cut -d"|" -f 3 | xargs)
+export LC_ALL="en_US.UTF-8"
+find $POOLPATH/boot-* -type f -mtime +5 -exec sh -c 'virsh vol-delete {} || rm {}' \;
```

Comment:
`sudo virsh vol-delete ..` ?

Reply:
sudo shouldnt be needed as long as user as proper permissions in the corresponding pool path

Reply:
But `virsh` was executed as root when making pools and volumes:

```
[kni@cnfdc1 ~]$ ls -alsh /storage/boot-f8b468dc-ddb8-43d0-9ba8-50f7470b80f0-iso-5664ed21-7354-45ff-aae6-e2f2f8198277.img 
445M -rw------- 1 qemu qemu 445M Oct  7  2021 /storage/boot-f8b468dc-ddb8-43d0-9ba8-50f7470b80f0-iso-5664ed21-7354-45ff-aae6-e2f2f8198277.img
[kni@cnfdc1 ~]$ virsh vol-delete /storage/boot-f8b468dc-ddb8-43d0-9ba8-50f7470b80f0-iso-5664ed21-7354-45ff-aae6-e2f2f8198277.img
error: failed to get vol '/storage/boot-f8b468dc-ddb8-43d0-9ba8-50f7470b80f0-iso-5664ed21-7354-45ff-aae6-e2f2f8198277.img', specifying --pool might help
error: Storage volume not found: no storage vol with matching path '/storage/boot-f8b468dc-ddb8-43d0-9ba8-50f7470b80f0-iso-5664ed21-7354-45ff-aae6-e2f2f8198277.img'

[kni@cnfdc1 ~]$ sudo virsh vol-delete /storage/boot-f8b468dc-ddb8-43d0-9ba8-50f7470b80f0-iso-5664ed21-7354-45ff-aae6-e2f2f8198277.img
Vol /storage/boot-f8b468dc-ddb8-43d0-9ba8-50f7470b80f0-iso-5664ed21-7354-45ff-aae6-e2f2f8198277.img deleted
```

Reply:
virsh wasn't executed as root, it's libvirt which puts qemu/qemu permissions to this volume
can you try to delete such a problematic image with
virsh vol-delete --pool $pool_name $basename_img

Reply:
Need to use `virsh -c qemu:///system` to remove it, by default it's `qemu:///session`